content/en/users/data/management/datahub/file-management/_index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: DataHub File Management
+linkTitle: File Management
+weight: 30
+type: docs
+aliases:
+  - /users/datahub/datahub-file-management
```

Comment:
What's the reasoning behind using this alias?

Reply:
I would say none in particular. Indeed it would correspond top the path where it will be. Do you prefer to remove it?

Reply:
So the location of the file and files will define the path of the URL, so currently it will be `/users/data/management/datahub/file-management`, which looks right.
The aliases are mainly meant to use when we re-organise the repository so that we don't have broken links, and the only thing they will do will be to redirect to the real URL.
So I don't think it's appropriate for a new file.

Reply:
I've removed it now